routes/feedback/feedback.njk

```diff
@@ -1,21 +1,15 @@
 {% extends "base.njk" %}
 
 {% block content %}
+    <a href="agreement-1">&#9666; {{  __("back") }}</a>
+    <h1>{{ __("post-service-feedback.title") }}</h1>
 
-    <h1>{{ __('feedback.title') }}</h1>
-
-    <div>
-    {% if notifySetup %}
```

Comment:
why take out the `{% if notifySetup %}`?

Reply:
Now the form data is sent to the feedback collector:
`action="https://cds-feedback-collector.herokuapp.com/en/send?id=c966af51-e0a3-412f-99fd-47411788da9f"`

and that sends an email through notify if you've configured it to there.

Reply:
nice.